pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -3782,6 +3817,236 @@ func (prt PipelineRunTest) reconcileRun(namespace, pipelineRunName string, wantE
 	return reconciledRun, clients
 }
 
+func TestReconcile_RemotePipelineRef(t *testing.T) {
```

Comment:
This test includes a lot of stuff not actually related to remote pipelines right? Resources and params and all that aren't really relevant to just exercising bundle fetching/parsing. Can we remove some of the noise and just have this run a simple one-task pipeline?